src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -0,0 +1,682 @@
+---
+title: Experimental fonts API
+sidebar:
+  label: Fonts
+i18nReady: true
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 6
+---
+
+import Since from '~/components/Since.astro';
+import { Steps, Tabs, TabItem } from '@astrojs/starlight/components';
+
+<p>
+
+**Type:** `FontFamily[]`<br />
+<Since v="5.7.0" />
+</p>
+
+This experimental feature allows you to use fonts from your filesystem and various providers through a unified API. It automatically creates optimized fallbacks and preload links:
+
+```js title="astro.config.mjs"
+import { defineConfig, fontProviders } from "astro/config"
+
+export default defineConfig({
+    experimental: {
+        fonts: [{
+            provider: fontProviders.google(),
+            name: "Roboto",
+            cssVariable: "--font-roboto",
+        }]
+    }
+})
+```
+
+```astro title="src/components/Head.astro"
+---
+import { Font } from 'astro:assets'
+---
+
+<Font family='--font-roboto' preload />
+
+<style>
+body {
+    font-family: var(--font-roboto);
+}
+</style>
+```
+
+## Usage guide
+
+To use the experimental fonts API you need to update your Astro configuration, call the `<Font />` component and use the CSS variable:
+
+### Astro configuration
+
+`experimental.fonts` accepts an array of `FontFamily`. To get started, let's setup one family:
+
+<Steps>
+
+1. Register an empty family to start with:
+
+    ```js title="astro.config.mjs" ins={4-8}
+    import { defineConfig } from "astro/config"
+
+    export default defineConfig({
+        experimental: {
+            fonts: [{
+                // We are going to add things here
+            }]
+        }
+    })
+    ```
+
+2. Pick a provider from the [built-in remote provider](#available-remote-font-providers) or the [local provider](#local-font-families). In this example, we'll use the Google font provider:
+
+    ```js title="astro.config.mjs" del={1} ins={2,7}
+    import { defineConfig } from "astro/config"
+    import { defineConfig, fontProviders } from "astro/config"
+
+    export default defineConfig({
+        experimental: {
+            fonts: [{
+                provider: fontProviders.google(),
+            }]
+        }
+    })
+    ```
+
+3. Pick a font family supported by your provider (documented for each provider in the [reference](#available-remote-font-providers)). For our example, choose one on the [Google Fonts website](https://fonts.google.com/):
```

Comment:
Since our example shows Roboto this should say "For our example we've chosen Roboto" with a link to the font

I'm on my phone or I would go grab the link and do a suggestion

Reply:
```suggestion
3. Pick a font family supported by your provider (documented for each provider in the [reference](#available-remote-font-providers)). For our example, we've choosen [Roboto from Google Fonts](https://fonts.google.com/specimen/Roboto):
```
Something like that?

Reply:
Looks great :)